<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleSearchWidget</class>
 <widget class="QWidget" name="SampleSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,100">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Sample</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>species:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>quality:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DBSelector" name="s_species"/>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="s_ffpe">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="s_tumor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>tumor:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ffpe:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DBSelector" name="s_name"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="s_quality"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Project</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DBSelector" name="p_name"/>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Processing system</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DBSelector" name="sys_name"/>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Run</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DBSelector" name="r_name"/>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="search_btn">
       <property name="text">
        <string>search</string>
       </property>
       <property name="icon">
        <iconset resource="GSvar.qrc">
         <normaloff>:/Icons/NGSD_search.png</normaloff>:/Icons/NGSD_search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DBTableWidget" name="sample_table"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBSelector</class>
   <extends>QLineEdit</extends>
   <header>DBSelector.h</header>
  </customwidget>
  <customwidget>
   <class>DBTableWidget</class>
   <extends>QTableWidget</extends>
   <header>DBTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GSvar.qrc"/>
 </resources>
 <connections/>
</ui>
